spec/factories/issue_factory.rb

```diff
@@ -0,0 +1,14 @@
+CODES = ("AAAA".."ZZZZ").to_a.shuffle
```

Comment:
I would maybe rename `CODES` to something more descriptive like unique strings, as it's not actually returning a code!

Reply:
Thank you for pointing that out! I've changed CODES to UNIQUE_STRINGS. It does make a lot more sense!